types/pg/index.d.ts

```diff
@@ -40,6 +36,7 @@ export interface PoolConfig extends ClientConfig {
       max?: number;
```

Comment:
But is it documented? I can't find it from document and example.

Reply:
Ah you're right I think I only saw it in the code.
Perhaps we rather avoid to add it, and sticks to max only which is the 'official' property since 7.0